pandas/tests/plotting/test_frame.py

```diff
@@ -1306,6 +1306,12 @@ def test_plot_scatter_with_c(self):
         float_array = np.array([0.0, 1.0])
         df.plot.scatter(x="A", y="B", c=float_array, cmap="spring")
 
+    def test_plot_scatter_with_s(self):
+        # this refers to GH 32904
+        df = DataFrame(np.random.random((10, 3)) * 100, columns=["a", "b", "c"],)
+
+        _check_plot_works(df.plot.scatter, x="a", y="b", s="c")
```

Comment:
It would be good to have some assertion here that the argument actually had an effect. Perhaps something like

```suggestion
        ax = df.plot.scatter(x="a", y="b", s="c")
        assert len(set(ax.collections[0].get_sizes())) > 1
```

Reply:
Actually, I'm confused about the correct way of using `versionchanged`.  I read https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html and I think this is suitable:
```
        s : str, scalar or array_like, optional
            The size of each point. Possible values are:

            .. versionchanged:: 1.1.0

            - A string with the name of the column to be used for marker's size.
```
However, not sure about the indentation and whether second argument to `versionchanged` is needed (searching pandas code shows that sometimes the description is added, but it's not everywhere).